pkg/resource/vpc_origin/hooks.go

```diff
@@ -0,0 +1,71 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package vpc_origin
+
+import (
+	"context"
+
+	svcapitypes "github.com/aws-controllers-k8s/cloudfront-controller/apis/v1alpha1"
+	util "github.com/aws-controllers-k8s/cloudfront-controller/pkg/resource/tags"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	svcsdk "github.com/aws/aws-sdk-go-v2/service/cloudfront"
+	svcsdktypes "github.com/aws/aws-sdk-go-v2/service/cloudfront/types"
+)
```

Comment:
nit: https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e
```suggestion
    "context"
   
	"github.com/aws/aws-sdk-go-v2/aws"
	svcsdk "github.com/aws/aws-sdk-go-v2/service/cloudfront"
	svcsdktypes "github.com/aws/aws-sdk-go-v2/service/cloudfront/types"
	
	svcapitypes "github.com/aws-controllers-k8s/cloudfront-controller/apis/v1alpha1"
	util "github.com/aws-controllers-k8s/cloudfront-controller/pkg/resource/tags"
```

Reply:
My Go formatter in VSCode seems to be grouping these imports differently. Is there a setting or config that I need to change?